pages/Getting Started/Installation.md

```diff
@@ -7,10 +7,10 @@ Hyprland.
 
 ### Distros
 
-Arch and NixOS are very supported. For any other distro (not based on Arch/Nix)
-you might have varying amounts of success. However, since Hyprland is extremely
-bleeding-edge, distros like Pop!\_OS, Ubuntu, etc. might have **major** issues
-running Hyprland.
+Arch, NixOS and openSUSE Tumbleweed are very supported. For any other distro
```

Comment:
why add suse here? very supported means someone to answer your question is online on the discord most of the time

Reply:
Dunno, I figured very supported means that there is package that is likely up to date (granted the AUR package is two versions behind; so maybe not) you can install and it should just work out of the box. That a bug is the cause of a packaging issue is rather unlikely (esp. with meson and statically linked wlroots) and should there be, a user can write me, or the other maintainers of the X11:Wayland project an email or post on the openSUSE users and/or support mailing list.

Reply:
fair